docs/docs-content/clusters/public-cloud/aws/eks-hybrid-nodes/architecture.md

```diff
@@ -0,0 +1,147 @@
+---
+sidebar_label: "Architecture"
+title: "Architecture"
+description: "Learn about the architecture used by Amazon EKS Hybrid Nodes when deployed with Palette."
+hide_table_of_contents: false
+tags: ["public cloud", "aws", "architecture", "eks hybrid nodes"]
+sidebar_position: 0
+---
+
+Palette enables importing and managing Amazon Elastic Kubernetes Service (Amazon EKS) Hybrid Nodes. For an overview of
+all necessary prerequisites, refer to
+[Prerequisite setup for hybrid nodes](https://docs.aws.amazon.com/eks/latest/userguide/hybrid-nodes-prereqs.html).
+
+These are some of the architectural highlights when using Palette to manage your Amazon EKS Hybrid Nodes.
+
+- Add Container Network Interface (CNI) layers to handle networking for hybrid nodes using affinity rules.
```

Comment:
@TylerGillson I removed the bullet point for the "Each cluster profile for a hybrid node pool includes
  the following configurable layers" bit.
  
  I then moved this sentence to here as I wanted to highlight how the CNI layer is configured for the EKS cluster in Palette to handle the hybrid nodes.
  
  I believe this is now correct, but checking with you to see whether we can highlight that or not.